### ISSUE TITLE
Add support for Shan language (shn)

### ISSUE THREAD

**Issue by ronaldaug** [tesseract-ocr/langdata#151]
Could someone help me to add the Shan language in tesseract? 

Shan language = [https://en.wikipedia.org/wiki/Shan_language](https://en.wikipedia.org/wiki/Shan_language)
Language code = shn 
Shan Wiki = [https://shn.wikipedia.org](https://shn.wikipedia.org)
All Shan words (including IPA) = [jsonfile](https://github.com/kwarm/shan-words/blob/master/jsonfile.json)
Websites that are using Shan scripts = [https://shannews.org/](https://shannews.org/) , [http://shanunicode.com/](http://shanunicode.com/)
Font = https://saosu-mp.github.io/font/PangLong/PangLong.ttf
Shan syllable break = [https://github.com/kwarm/syllable-break](https://github.com/kwarm/syllable-break)

Some Shan characters such as `င သ တ ထ ပ မ ယ ရ လ ဝ ႉ း ွ ု ူ ိ ီ ် ၊ ။` are similar to [Myanmar (Burmese)](https://github.com/tesseract-ocr/langdata/tree/master/mya).

Thanks in advance

**Comment by ronaldaug** [comment]
It seems this repo isn't active or maintained.

**Comment by stweil** [comment]
That's correct, this repo is for the old Tesseract 3.05 and the legacy OCR recognizer.
The more recent repository is https://github.com/tesseract-ocr/langdata_lstm. Should I move this issue to that repo?

**Comment by ronaldaug** [comment]
Yes please, thanks @stweil .
